pkg/cronjob/executor.go

```diff
@@ -0,0 +1,164 @@
+package cronjob
+
+import (
+	"context"
+	"fmt"
+
+	testsv3 "github.com/kubeshop/testkube-operator/api/tests/v3"
+	testsuitesv3 "github.com/kubeshop/testkube-operator/api/testsuite/v3"
+	"github.com/kubeshop/testkube/pkg/api/v1/testkube"
+	"github.com/kubeshop/testkube/pkg/cloud"
+	testsmapper "github.com/kubeshop/testkube/pkg/mapper/tests"
+	testsuitesmapper "github.com/kubeshop/testkube/pkg/mapper/testsuites"
+	cronjobtcl "github.com/kubeshop/testkube/pkg/tcl/testworkflowstcl/cronjob"
+	"github.com/kubeshop/testkube/pkg/testworkflows/testworkflowexecutor"
+	"github.com/kubeshop/testkube/pkg/workerpool"
+)
+
+const (
+	concurrencyLevel = 10
+)
+
+func (s *Scheduler) executeTestWorkflow(ctx context.Context, testWorkflowName string, cron *testkube.TestWorkflowCronJobConfig) {
+	request := &cloud.ScheduleRequest{
```

Comment:
Not sure about edge case - shouldn't it check the last execution? Otherwise each service re-initialization will i.e. reconsider again i.e. `* * * * *` cronjob

Reply:
i can try to reproduce, tif it happens